packages/@aws-cdk/aws-kinesisanalytics-flink/lib/application.ts

```diff
@@ -840,6 +874,27 @@ export interface ApplicationProps {
    * @default CDK's default LogGroup
    */
   readonly logGroup?: logs.ILogGroup;
+
+  /**
+   * Deploy the Flink application in a VPC.
+   *
+   * @default no VPC
```

Comment:
```suggestion
   * @default - no VPC
```

Reply:
I just pushed a change which adds hyphens for these kinds of @default phrases to the entire file.